mqtt/edgelet/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "edgelet"
```

Comment:
should we call it "edgelet-client" instead?

Reply:
yeah, it makes sense. it's hard to misinterpret `edgelet` workspace with `edgelet` client though.